SlackAPI/SlackClient.cs

```diff
@@ -645,5 +645,16 @@ public void UploadFile(Action<FileUploadResponse> callback, byte[] fileData, str
                 callback(result.Deserialize<FileUploadResponse>());
             }
         }
+
+        public void DeleteFile(Action<FileDeleteResponse> callback, string file = null)
+        {
+            List<Tuple<string, string>> parameters = new List<Tuple<string, string>>();
+
+            if (!string.IsNullOrEmpty(file))
+                parameters.Add(new Tuple<string, string>("file", file));
+            else return;
+
+            APIRequestWithToken(callback, parameters.ToArray());
```

Comment:
This should be simplified to just pass in the parameter as is:
`APIRequestWithToken(callback, new Tuple<string, string>(...))

Reply:
Doesn't APIRequestWithToken expect an Array of Tuple<>?

Reply:
It does, but not exactly, that parameter is prefixed with `params` meaning you can just pass single objects by themselves,

see: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params